src/__tests__/patients/allergies/AllergiesList.test.tsx

```diff
@@ -16,52 +15,38 @@ describe('Allergies list', () => {
     jest.spyOn(PatientRepository, 'find').mockResolvedValueOnce(mockPatient)
     const history = createMemoryHistory()
     history.push(`/patients/${mockPatient.id}/allergies`)
-    let wrapper: any
     await act(async () => {
-      wrapper = await mount(
+      await render(
         <Router history={history}>
           <AllergiesList patientId={mockPatient.id} />
         </Router>,
       )
     })
-
-    wrapper.update()
-    return { wrapper: wrapper as ReactWrapper, history }
+    return { history }
   }
 
   it('should render a list of allergies', async () => {
     const expectedAllergies = [{ id: '456', name: 'some name' }]
-    const { wrapper } = await setup(expectedAllergies)
-    const listItems = wrapper.find(ListItem)
-
-    expect(wrapper.exists(List)).toBeTruthy()
+    await setup(expectedAllergies)
+    const listItems = screen.getAllByRole('button')
     expect(listItems).toHaveLength(expectedAllergies.length)
-    expect(listItems.at(0).text()).toEqual(expectedAllergies[0].name)
+    expect(screen.getByText(expectedAllergies[0].name)).toBeInTheDocument()
   })
 
   it('should display a warning when no allergies are present', async () => {
     const expectedAllergies: Allergy[] = []
-    const { wrapper } = await setup(expectedAllergies)
-
-    const alert = wrapper.find(Alert)
-
-    expect(wrapper.exists(Alert)).toBeTruthy()
-    expect(wrapper.exists(List)).toBeFalsy()
-
-    expect(alert.prop('color')).toEqual('warning')
-    expect(alert.prop('title')).toEqual('patient.allergies.warning.noAllergies')
-    expect(alert.prop('message')).toEqual('patient.allergies.addAllergyAbove')
+    await setup(expectedAllergies)
+    expect(screen.getByText('patient.allergies.warning.noAllergies')).toBeInTheDocument()
+    expect(screen.getByText('patient.allergies.addAllergyAbove')).toBeInTheDocument()
```

Comment:
is this working?

Reply:
Can you please elucidate? If the passing tests mean it's working, then yes. I don't entirely get what you are trying to say.

Reply:
elucidate => clarify

Reply:
@balavishnuvj what about it are you concerned about?

Reply:
`'patient.allergies.warning.noAllergies'` is this a text what user would be seeing... earlier this used to be prob. so just clarifying

Reply:
The translation keys are probably missing (some warnings are shown when tests run), so it is falling back to rendering `'patient.allergies.warning.noAllergies'` in the dom. I used `screen.debug` and saw this in the output. It should probably be dealt with. Should a new issue be created? @balavishnuvj  @JacobMGEvans

Reply:
For now, we are focusing on conversion. Get the RTL to work, rip out enzyme, then we can go back and improve or add tests and helpers as needed.

Reply:
Some things also won't be ideal RTL practices or "look right" which is fine too. I had to get a little creative in the Nav conversion lol 😆